test/clj_kondo/jdbc_test.clj

```diff
@@ -0,0 +1,14 @@
+(ns clj-kondo.jdbc-test
+  (:require
+   [clj-kondo.test-utils :refer [lint!]]
+   [clojure.java.io :as io]
+   [clojure.test :as t :refer [deftest is testing]]
+   [missing.test.assertions]))
+
+(deftest next-jdbc-test
```

Comment:
Instead of only testing the avoidance of false positive, it might also be nice to test if some of  the findings you emitted work? E.g. not a symbol as a binding.

Reply:
The way I usually test "findings" is to first call (prn (lint! "...")) and then paste that output back into the test file and test it with (assert-submaps the-output (lint! "..."))

Reply:
I wasn't sure how you handled those tests in the suite. `corpus/jdbc/next.clj` has several of those tests that I used to manually verify the behaviors. Should I just add that into this test with verification of the four expected findings?

Reply:
Yeah. For example start looking here: https://github.com/borkdude/clj-kondo/blob/master/test/clj_kondo/main_test.clj#L14
and scan that file for corpus. It contains lots of examples of how this is tested.